wideq/core.py

```diff
@@ -129,6 +129,8 @@ def lgedm_post(url, data=None, access_token=None, session_id=None):
                 raise NotLoggedInError()
             elif code == "0106":
                 raise NotConnectedError()
+            elif code == "0010": #I don't remember exactly but it's not critical error
+                return out
```

Comment:
It would be great to investigate a little more to understand what's going on in this case. Should we be issuing a warning?